client/loops.lua

```diff
@@ -15,3 +16,18 @@ CreateThread(function()
         end
     end
 end)
+
+CreateThread(function()
+    while true do
+        local PauseMenuActive = IsPauseMenuActive()
+
+        if PauseMenuActive and not Active then
+            Active = true
+            TriggerEvent('qbx_core:client:onPauseMenuActive', Active)
+        elseif Active and PauseMenuActive then
+            Active = false
+            TriggerEvent('qbx_core:client:onPauseMenuActive', Active)
+        end
+        Wait(1000)
+    end
+end)
```

Comment:
Use https://overextended.dev/ox_lib/Modules/Cache/Shared#adding-new-cached-values and lib.onCache to do the event handling.

Reply:
I don't think they'll add a new cache option for something that's just true or false. I don't see a real use for us making a new cached value for it either